app/namespace.go

```diff
@@ -269,6 +270,32 @@ func (c *NamespaceClient) createNamespace(n *namespace.Namespace, p []*auth.User
 	return PrintProto(res)
 }
 
+func (c *NamespaceClient) addReplica(ctx *cli.Context) error {
+	var resourceVersion string
+	if v := ctx.String(ResourceVersionFlagName); v != "" {
+		resourceVersion = v
+	}
+
+	region := ctx.String(namespaceRegionFlagName)
```

Comment:
[nit] is there a helper you could call here with predefined error?

Reply:
I will check